package.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
it's needed for grunt, I've had to add this back a few times because it never got checked back in.